src/gm3/components/catalog/legend.js

```diff
@@ -1,10 +1,10 @@
 /*
  * The MIT License (MIT)
  *
- * Copyright (c) 2016-2017 Dan "Ducky" Little
+ * Copyright (c) 2016-2017 Dan 'Ducky' Little
  *
  * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
+ * of this software and associated documentation files (the 'Software'), to deal
```

Comment:
Why do some of these use ' and others use "  ?

Reply:
"%s/"/'/" has caught the license text.